src/main/scala/com/databricks/labs/overwatch/ApiMeta.scala

```diff
@@ -0,0 +1,98 @@
+package com.databricks.labs.overwatch
+import org.apache.log4j.{Level, Logger}
+
+/**
+ * Configuration for each API.
+ */
+trait ApiMeta {
+
+  private val logger: Logger = Logger.getLogger(this.getClass)
+  var paginationKey:String="";
+  var paginationToken:String="";
+  var dataframeColumns:String="";
+  var apiCallType:String="";
```

Comment:
perfect example, don't default this to something that will never make sense...either initialize it to null `_` or something that is "normal" like "GET"